src/main.js

```diff
@@ -1,27 +0,0 @@
-import { createApp } from 'vue'
-import App from './App.vue'
-import router from './router'
-import { createStore } from 'vuex'
-
-const store = createStore({
-  state() {
-    return {
-      times: [],
-      fromMeToGmail: [],
-      fromMeToNonGmail: [],
-      toMeFromGmail: [],
-      toMeFromNonGmail: []
-    }
-  },
-  mutations: {
-    setStats(state, payload) {
-      state.times = payload.times
-      state.fromMeToGmail = payload.fromMeToGmail
-      state.fromMeToNonGmail = payload.fromMeToNonGmail
-      state.toMeFromGmail = payload.toMeFromGmail
-      state.toMeFromNonGmail = payload.toMeFromNonGmail
-    }
-  }
```

Comment:
Very nice setup :) All you're missing is an action to fetch your emails and mutate the store using `setStats`.